README.md

```diff
@@ -104,6 +104,7 @@ For enhanced file previews (with `scope.sh`):
 * `mediainfo` or `exiftool` for viewing information about media files
 * `odt2txt` for OpenDocument text files (`odt`, `ods`, `odp` and `sxw`)
 * `python` or `jq` for JSON files
+* `jupyter nbconvert` for Jupyter Notebooks
```

Comment:
Is nbconvert a separate command? Otherwise I think listing it as just `jupyter` is nicer, since that's what you'd want to install.

Reply:
It seems that 2 packages can do the same things, one need Jupyter, not the other :

```
jupyter-nbconvert/stable,stable 5.6.1-3 all
  Jupyter notebook conversion  –⋅scripts
  Depends: python3-nbconvert (= 5.6.1-3), python3:any, jupyter-core

python3-nbconvert/stable,stable 5.6.1-3 all
  Jupyter notebook conversion (Python 3)
  Depends: python3-bleach, python3-defusedxml, python3-entrypoints, python3-jinja2, python3-jupyter-core, python3-mistune, python3-nbformat, python3-pandocfilters (>= 1.4), python3-pygments, python3-testpath, python3-traitlets, python3:any
```

Reply:
Ah, so nbconvert *is* something that is installed in addition to the regular Jupyter? I.e., it's not like `git log` which is a subcommand included with git?

Reply:
It can also be installed via pip: https://nbconvert.readthedocs.io/en/latest/install.html. I think if you install jupyter it will very likely also install nbconvert, and is probably more clear than trying to guess the path to the python environment that has nbconvert installed. So IMHO calling it as jupyter notebook makes the most sense, but there might be a better solution.

Reply:
This is fine, since it doesn't necessarily come with Jupyter we actually want to mention the subcommand.